File: android-v0-compose/app/src/main/java/com/example/v0clone/data/model/RegionDictionaryModels.kt
package com.xlwl.AiMian.data.model

import com.google.gson.annotations.SerializedName

data class RegionDictionaryItem(
    val id: String,
    val code: String?,
    val name: String,
    val level: Int,
    val parentId: String?,
    val sortOrder: Int = 0,
    @SerializedName("isActive")
    val isActive: Boolean = true,
    @SerializedName("children")
    val children: List<RegionDictionaryItem> = emptyList()
)

File: iOS-v0/Sources/Models/RegionDictionaryModels.swift
import Foundation

/// 地区字典 - 对齐 Android RegionDictionaryItem
struct RegionDictionaryItem: Codable, Identifiable, Hashable, Sendable {
  let id: String
  let code: String?
  let name: String
  let level: Int
  let parentId: String?
  let sortOrder: Int
  let isActive: Bool
  let children: [RegionDictionaryItem]
}
